backend/apps/innogram/src/chat/dto/init-chat.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString } from 'class-validator';
+
+export class InitChatDto {
+  @IsNotEmpty()
+  @IsString()
+  @ApiProperty()
```

Comment:
```suggestion
  @ApiProperty({
    description: 'blabla',
    example: 1
  })
```
you can add make specific documentation like this

Reply:
done